src/project/internal/projectmigrator.cpp

```diff
@@ -199,3 +200,56 @@ bool ProjectMigrator::resetAllElementsPositions(mu::engraving::MasterScore* scor
     score->setResetAutoplace();
     return true;
 }
+
+void ProjectMigrator::doPostLayoutMigrationIfNeeded(engraving::EngravingProjectPtr project)
+{
+    mu::engraving::MasterScore* score = project->masterScore();
+
+    bool needRelayout = false;
+
+    if (relayoutUserModifiedCrossStaffBeams(score)) {
+        needRelayout = true;
+    }
+    // As we progress, likely that more things will be done here
+
+    if (needRelayout) {
+        score->update();
+    }
+}
+
+bool ProjectMigrator::relayoutUserModifiedCrossStaffBeams(engraving::MasterScore* score)
+{
+    bool found = false;
```

Comment:
I'm failing to see where this variable is read

Reply:
lol, obviously I meant `return found;`